client/reader/sidebar/index.jsx

```diff
@@ -172,18 +172,21 @@ export class ReaderSidebar extends Component {
 				<QueryReaderTeams />
 				<QueryReaderOrganizations />
 
+				<li className="sidebar-header">
+					<h3>{ translate( 'Reader' ) }</h3>
+					<p>{ translate( 'Keep up with your interests.' ) }</p>
+				</li>
+
 				<SidebarItem
 					label={ translate( 'Search' ) }
 					onNavigate={ this.handleReaderSidebarSearchClicked }
-					customIcon={ <ReaderSearchIcon viewBox="-3 0 24 24" /> }
```

Comment:
Thoughtout the reader sidebar we were using `-3` so I have replaced it with `0` as it was not necessary and also in "Sites" we don't use this trickt so using `0` also make it consistent.